packages/zwave-js/src/lib/node/mixins/05_Security.ts

```diff
@@ -77,4 +77,15 @@ export abstract class NodeSecurityMixin extends NetworkRoleMixin
 		// If we don't have the info for every security class, we don't know the highest one yet
 		return missingSome ? NOT_KNOWN : SecurityClass.None;
 	}
+
+	/**
+	 * Should be `true` when an S2-capable node fails to be included when S2 is expected.
+	 */
+	public get skipInterview(): boolean {
```

Comment:
I think `failedS2Bootstrapping` or something like that would capture the intention better.